src/main/java/eu/dissco/backend/service/AnnotationService.java

```diff
@@ -205,10 +205,10 @@ public boolean tombstoneAnnotation(String prefix, String suffix, Agent agent, bo
     Optional<Annotation> result;
     if (isAdmin) {
       log.info("Admin tombstoning annotation {}", id);
-      result = repository.getActiveAnnotation(id);
+      result = repository.getActiveAnnotation(id, null);
```

Comment:
hmm not sure about passing null, why not do the isAdmin and then check on `!isAdmin` instead of the null check. Anyway this is not wrong

Reply:
if `isAdmin=true` then the userId is going to be null anyway, thought this was a bit more concise. 
The alternative is to have two functions but someone suggested only having one :)